e2e-test/resources/teaspoons/teaspoons_gcp_set_up_control_workspace.py

```diff
@@ -129,6 +126,11 @@ def workspace_exists(billing_project_name, workspace_name, rawls_url, token):
         logging.info(f"sharing workspace wtih {email_to_share_with}")
         share_workspace_grant_owner(firecloud_orch_url, billing_project_name, workspace_name, email_to_share_with, user_token)
 
+# set TTL on files in submissions directory of workspace bucket
+ttl_days = 14  # delete after 2 weeks
+matches_prefix_list = ["submissions/"]
```

Comment:
i think we also want to add the `user-input-files` prefix as well.

Reply:
ah great catch, thanks!